core/.js/src/main/scala/io/udash/utils/FileService.scala

```diff
@@ -0,0 +1,45 @@
+package io.udash.utils
+
+import java.io.IOException
+
+import org.scalajs.dom._
+import scalatags.JsDom.all._
+
+import scala.scalajs.js
+import js.JSConverters._
+import js.typedarray._
+
+import scala.concurrent._
+import scala.util.Try
+
+object FileService {
```

Comment:
Could be tested in `guide-selenium` I guess.

Reply:
I have no idea how to test file uploading via selenium :(